lib/gitx/github.rb

```diff
@@ -140,7 +140,7 @@ def username
     #   https://github.com/wireframe/gitx.git #=> wireframe/gitx
     def github_slug
       remote = repo.config['remote.origin.url']
-      remote.to_s.gsub(/\.git$/,'').split(/[:\/]/).last(2).join('/')
+      remote.to_s.gsub(/\.git$/, '').split(/[:\/]/).last(2).join('/')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.